ovh/import_hosting_privatedatabase_user_grant_test.go

```diff
@@ -0,0 +1,61 @@
+package ovh
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+const testAccHostingPrivateDatabaseUserGrantImportBasic = `
+resource "ovh_hosting_privatedatabase_user_grant" "grant" {
+    service_name  = "%s"
+    user_name     = "%s"
+    database_name = "%s"
+    grant         = "%s"
+}
+`
+
+func TestAccHostingPrivateDatabaseUserGrant_importBasic(t *testing.T) {
+	resourceName := "ovh_hosting_privatedatabase_user_grant.grant"
```

Comment:
You need to add new environment variable in the documentation too:
https://registry.terraform.io/providers/ovh/ovh/latest/docs#testing-and-development